frontend/src/components/quizzerhub_components/quiz_card.tsx

```diff
@@ -7,6 +7,20 @@ interface Props {
     quiz: Quiz
     selectQuiz: (id: string) => void
 }
+export const getColorByAvgScore = (score: number) => {
```

Comment:
picky: a switch statement could be used here

Reply:
i wasn't sure how to do it but do you mean like this?
```
switch (true) {
    case score > 85:
        return 'green';
    case score > 70:
        return 'blue';
    case score > 50:
        return 'yellow';
    case score > 30:
        return 'orange';
    default:
        return 'red';
}
```

Reply:
yep

Reply:
ah i see, actually i think you can ignore my comment on the switch case then, didnt properlly read what the conditionals were doing